app/views/layouts/_phase_banner.html.erb

```diff
@@ -1,10 +1,12 @@
-<div class="govuk-phase-banner">
+<aside role="complementary" aria-labelledby="phase_banner" class="govuk-width-container">
+  <div class="govuk-phase-banner">
     <p class="govuk-phase-banner__content">
-    <strong class="govuk-tag govuk-phase-banner__content__tag">
+      <strong class="govuk-tag govuk-phase-banner__content__tag">
         Beta
-    </strong>
-    <span class="govuk-phase-banner__text">
-        This is a new service – your <a class="govuk-link" target="_blank" rel="noopener noreferrer" href="https://docs.google.com/forms/d/e/1FAIpQLSeLpjkKhcwKUjBKRx29fUgzgslSAkE9pPzkcmp3Oer5T6JHDw/viewform">feedback</a> will help us to improve it.
-    </span>
+      </strong>
+      <span class="govuk-phase-banner__text">
+          This is a new service – your <a class="govuk-link" target="_blank" rel="noopener noreferrer" href="https://docs.google.com/forms/d/e/1FAIpQLSeLpjkKhcwKUjBKRx29fUgzgslSAkE9pPzkcmp3Oer5T6JHDw/viewform">feedback</a> will help us to improve it.
+      </span>
     </p>
-</div>
\ No newline at end of file
+  </div>
+</aside>
```

Comment:
I reformatted this and put the first an last lines in the partial to DRY up the three places that it is included